Tests/Factory/DiactorosFactoryTest.php

```diff
@@ -161,4 +161,33 @@ public function testCreateResponseFromBinaryFile()
 
         $this->assertEquals('Binary', $psrResponse->getBody()->__toString());
     }
+
+    public function testUploadErrNoFile()
+    {
+        $file = new UploadedFile('', '', null, 0, UPLOAD_ERR_NO_FILE, true);
+        $this->assertEquals(0, $file->getSize());
+        $this->assertEquals(UPLOAD_ERR_NO_FILE, $file->getError());
+        $this->assertFalse($file->getSize());
```

Comment:
There should be [a third argument](https://github.com/symfony/psr-http-message-bridge/pull/10#discussion_r58193053) like this:

```php
$this->assertFalse($file->getSize(), 'SplFile::getSize() returns false on error');
```